apps/codeforafrica/src/components/Projects/Projects.js

```diff
@@ -57,7 +56,9 @@ const Projects = React.forwardRef(function Projects(
   }, [data]);
 
   useEffect(() => {
-    router.push(queryParams, undefined, {
+    const { pathname } = window.location;
+    const url = pathname ? `${pathname}${queryParams}` : queryParams;
```

Comment:
```suggestion
    const url = `${pathname}${queryParams}`;
```

`pathname` should always be available.